plugin/src/PatchTree.lua

```diff
@@ -128,6 +137,8 @@ function Tree:addNode(parent, props)
     return node
 end
 
+-- Given a list of ancestor ids in descending order, builds the nodes for them
+-- using the patch and instanceMap info
 function Tree:buildAncestryNodes(ancestry, patch, instanceMap)
```

Comment:
I would not have guessed `ancestry` meant a list of ancestor IDs. `ancestorIds: { number }` would be clearer and removes the need for the "what" comments.